velox/substrait/tests/VeloxSubstraitRoundTripTest.cpp

```diff
@@ -402,6 +402,16 @@ TEST_F(VeloxSubstraitRoundTripTest, topNTwoKeys) {
       "SELECT * FROM tmp WHERE c0 > 15 ORDER BY c0 NULLS FIRST, c1 DESC NULLS LAST LIMIT 10");
 }
 
+TEST_F(VeloxSubstraitRoundTripTest, in) {
+  auto vectors = makeVectors(10, 4, 2);
+  createDuckDbTable(vectors);
+  auto plan = PlanBuilder(pool_.get())
+                  .values(vectors)
+                  .filter("c0 in (1, 2, 3)")
```

Comment:
Let's add tests for all supported types of IN values.

Reply:
> Current code goes to prestosql, and more types are supported in the IN function under sparksql. Do we need to use that?

It would be nice to test with both.

Reply:
> For bigint and string, an error in duckdb parser is found at [DuckParser.cpp](https://github.com/facebookincubator/velox/blob/main/velox/duckdb/conversion/DuckParser.cpp#L373). The reason is the parser adds some casting to generate a literal of certain type.

Maybe this can be fixed (in a separate PR).

Reply:
> It would be nice to test with both.

Tests were added for IN of presto and spark sql function.

> Maybe this can be fixed (in a separate PR).

Added TODO in this PR. More debugging is needed for the conversion from string to expression lists in DuckDB parser.

Reply:
> Added TODO in this PR. More debugging is needed for the conversion from string to expression lists in DuckDB parser.

Sounds good. Would you file an issue to explain what the problem is and maybe provide a small repro?